### DISCUSSION TITLE
Paid UI components/templates to get support for project

### DISCUSSION THREAD

**Discussion by Cijin** [franken-ui/ui#39] [Ideas]
Love the project. Have been looking for something like this for ages. Appreciate all the work that you are putting into it.

Was wondering if you have considered creating a set of paid components like [TailwindUI](https://tailwindui.com/) or [Alpine components](https://alpinejs.dev/components). I only mention it as I've paid for and got companies that I've worked for before to pay for them too. It's been quite challenging to convince the companies that I've worked for in Asia to support open source projects.

I think the work you are doing is quite valuable, as all the focus is on components for Js Frameworks.

Would love to see this project flourish in the long term. It has to be quite the undertaking to work on this project.

**Comment by sveltecult** [comment]
This is against my principle. I believe Franken UI should be FREE for all. Instead, I encourage donations if this project brought value/time-saving for someone or an organization.

https://github.com/sponsors/sveltecult
